Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2026D49.xml

```diff
@@ -38,6 +38,7 @@
     <Include ref='Geometry/TrackerSimData/data/PhaseII/TiltedTracker404/trackersens.xml'/>
     <Include ref='Geometry/TrackerSimData/data/PhaseII/TiltedTracker404/pixelsens.xml'/>
     <Include ref='Geometry/TrackerRecoData/data/PhaseII/TiltedTracker613_MB_2019_04/trackerRecoMaterial.xml'/>
+    <Include ref='SimTracker/TrackerMaterialAnalysis/data/trackingMaterialGroups_ForPhaseII.xml'/>
```

Comment:
why is this done only for geometry D49?
There are several active geometries in phase-2. At the bare minimum I would support also the geometries which feature the new Tracker geometries T21, T22 and T23, which are D63,D64 and D65

Reply:
That can be done as well, let's see if we can agree on a solution before extending changes to further geometries. I agreed on submitting a PR to see how the changes "would look like"